web/src/main/java/org/springframework/security/web/util/matcher/IpAddressMatcher.java

```diff
@@ -46,15 +47,18 @@
 	 */
 	public IpAddressMatcher(String ipAddress) {
 
+		String ipAddr = ipAddress;
```

Comment:
While this may be an improvement, @clementkng, let's please leave renaming the variable for another PR, unless there is a reason it's necessary?

Reply:
@jzheaux I changed it because ‘ipaddress’ is overwritten if we parse a mask, meaning that I can’t use it in a meaningful error message (to show the ip and mask together).

Reply:
I guess an alternative would be to rewrite the message in a way i.e. `IP address <address> too short for bitmask <mask>.` Let me know what you think.

Reply:
I agree that a message with placeholders is a good solution. I like it because of the smaller change footprint.

Reply:
@jzheaux I changed the message and changed the variables back.